auditlog_security/security/res_groups.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="group_can_view_audit_logs" model="res.groups">
+        <field name="name">View Audit Logs</field>
+    </record>
+</odoo>
```

Comment:
this group is not used  any unnecessary given you'll assign groups in the access rules record I think?

Reply:
@ntsirintanis The `View Audit Logs` group is still being used in the Inuka instances. 
Currently, it has around 11 users. 
It's also being used in approx. 21 record rules. 
This prevents to module from being upgraded.

Reply:
the rules from your legacy module should be deleted anyways, as you don't need them any more with this approach. for easier transition, mark them as noupdate in your legacy code, then those records won't be touched on module upgrade